Documentation/EN/Reference/Hooks/Index.rst

```diff
@@ -260,6 +261,50 @@ Implement the methods required by the interface:
         }
     }
 
+.. _selectorwidget_en:
+
+Hooks for the selector widget
+"""""""""""""""""""""""""""""
+
+There is a hook into the selector widget of the list view. It is executed just
+before the template gets rendered to HTML, if the selector widget is activated.
```

Comment:
I propose rearranging the sentence:

If the selector widget is activateed, the hook is executed …